packages/loaders/code-file/src/index.ts

```diff
@@ -57,11 +61,21 @@ function createGlobbyOptions(options: CodeFileLoaderOptions): GlobbyOptions {
  * Supported extensions include: `.ts`, `.tsx`, `.js`, `.jsx`, `.vue`
  */
 export class CodeFileLoader implements Loader<CodeFileLoaderOptions> {
+  private config: CodeFileLoaderConfig;
+  constructor(config?: CodeFileLoaderConfig) {
+    this.config = config ?? {};
+  }
```

Comment:
This allows applying config within codegen when we construct the loader vs for each file, where we don't know what loader we actually have.

Reply:
see https://github.com/dotansimha/graphql-code-generator/pull/6267/files#diff-c94057671bb6f895d718f875ae504bbc1180aaaf468b67a4678ba0923a55db96R80-R85